src/lib/validate.rs

```diff
@@ -0,0 +1,43 @@
+use std::process::exit;
+
+use crate::lib::configparser::*;
+use itertools::Itertools;
```

Comment:
what's itertools do? doesnt look like the file is using anything other than base rust iter stuff?

Reply:
Itertools is providing `Iter::partition_result()` for [validate.rs#L27](https://github.com/osusec/beavercds-ng/blob/dr/cmd-dispatch/src/lib/validate.rs#L27).